test1.py

```diff
@@ -1,88 +1,176 @@
-# weight_list = login2.get_req('weightentry')
-# print(weight_list)
-
-
-#ex2
+import random
+from pprint import pprint
 
-# for i in range(7):
-#     req1 = login2.create_nutrition_plan()
-#     print(req1)
-#
-# req1 = login2.get_req('nutritionplan')
-#
-# for nutrition in req1["results"][0:3]:
-#
-#     for i in range(5):
-#         req2 = login2.create_meals_for_nutrition_plans(nutrition['id'])
-#         print(req2)
-#
-#
-# for nutrition in req1["results"][3:7]:
-#     for i in range(5):
-#         req2 = login2.create_meals_for_nutrition_plans(nutrition['id'])
-#         print(req2)
-
-
-#ex3
-
-
-#ex4
-name_of_workout = "Workout5"
-add_workout = login2.add_workout_day(name_of_workout,"This is workout5")
-print(add_workout)
-# create days for the added workout
-
-get_workout = login2.get_req('workout')
-for workout in get_workout['results']:
-    if workout['name'] == name_of_workout:
-        id1 = workout['id']
-        add_day1 = login2.add_day(workout['id'],"Push",1)
-        add_day2 = login2.add_day(workout['id'],"Pull",2)
-        add_day3 = login2.add_day(workout['id'],"Legs",3)
-
-        days = login2.get_req('day')
-        list_of_days = list()
-        for day in days["results"]:
-            if day["training"] == id1:
-                list_of_days.append(day["id"])
+from main import WengerApi
 
-        for id in list_of_days:
+login1 = WengerApi("balbaealexandru@gmail.com","12345678abc","Token 5c4f8bef9dd4e6d72f0868971ac142cca0157431")
+
+
+def exercise1():
+    print("Create weight entry")
+    login1.create_weight_entry('2021-05-31','150')
+    print("Get weight entry")
+    weight_list = login1.get_req("weightentry")
+    print(weight_list)
+
+
+def exercise2():
+    login1.delete_nutrition_plans()
+
+    days_of_week = ['Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday', 'Sunday']
+    for day in days_of_week:
+        login1.create_nutrition_plan(day)
+
+
+    nutrition_plans = login1.get_req('nutritionplan')
+
+    for nutrition in nutrition_plans.get("results"):
+        req2 = login1.create_meals_for_nutrition_plans(nutrition.get('id'))
+        print(req2)
+
+    list_of_total_meals = list()
+
+    meals1 = login1.get_req('meal')
+    print(meals1)
+
+    for meal in meals1.get('results'):
+        list_of_total_meals.append(meal)
+
+    amount = 100
+    ingredients_req = login1.get_req('ingredient')
+    list_of_ingredients = ingredients_req.get('results')
+    for nutrition in nutrition_plans.get('results'):
+        for meal in list_of_total_meals:
+            kcal = 0
+            if meal.get('plan') == nutrition.get('id'):
+                if nutrition.get('description') in ['Monday', 'Tuesday', 'Wednesday']:
+                    while True:
+                        ingredient = random.choice(list_of_ingredients)
+                        kcal += ingredient.get('energy') * (amount/100)
+                        if kcal > 1750:
+                            print(kcal)
```

Comment:
Debug statements should not be present when you open a PR(unless they are used for a logging purpose).